docs/CQCToFile.rst

```diff
@@ -0,0 +1,36 @@
+Writing Commands to File with CQCToFile
+=======================================
+
+If you want to write the commands to a file instead of sending them via
+a server you can use :meth:`cqc.pythonLib.CQCToFile`. This class replaces
+:meth:`cqc.pythonLib.CQCConnection`. 
+
+:meth:`CQCToFile` has the following keyword arguments:
+
+- :code:`filename`: The file to which the commands will be written
+- :code:`pend_messages`: Whether to pend_messages until :code:`flush()` or to send them immediately
+- :code:`overwrite`: Whether to overwrite the file if there is already a file 
+with the name given by :code:`filename`. If this is :code:`False` and there
+is a file with the same name, a number will be appended to the name.
+
+The commands will be written to the file in textform. There will also be
+another file with the same name, but with 'binary' appended, which will contain
+the commands in binary form (which is easier to process later).
+
+:meth:`cqc.pythonLib.CQCToFile` does not support sending or receiving 
```

Comment:
Are there other commands not supported, like get-creation-time etc?

Reply:
I added sendGetTime(), allocate_qubits(), sendFactory(), create_qubits(),
tomography(), test_preparation() to the list. 

There are some more methods in CQCConnection which are not available in CQCToFile, but they are things that usually only get used internally, I think, such as startClassicalServer() and some others.